### DISCUSSION TITLE
Immortal Pear & Lady Bug

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1975] [Art]
Calling all fruit eaters!

We are looking for some community driven art for our upcoming Fruit Feature 🚀 

https://github.com/sunflower-land/sunflower-land/discussions/1949

For this discussion, we will be looking for contributions for the 'Immortal Pear' and the 'Lady Bug'.

**Requirements**

- Immortal Pear fits inside of Trophy case. See https://github.com/sunflower-land/sunflower-land/discussions/1893
- Lady Bug can be placed on a 1x1 square grid (16px wide & 20px high)
- Fits game aesthetic and colours
- Original pieces of work
- By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service.

**Submissions**

When submitting art, please include the following:

- Your farm ID (we will make sure the artists get some goodies 🏆 )
- A minified version of your art piece
- An example of the art placed in the game.

**Comment by MrSnorch** [comment]
Immortal Pear

Farm ID - **14106**
![груша1](https://user-images.githubusercontent.com/104712725/210695858-9efc888c-a31d-4bc4-9c03-cb3ba03de5a8.png)
![груша2](https://user-images.githubusercontent.com/104712725/210695876-219b1ca5-060d-442d-94ba-3da71f800467.png)
![груша1-export](https://user-images.githubusercontent.com/104712725/210700443-ca9172c1-774a-4962-939b-b355d3050a16.png)
![прев](https://user-images.githubusercontent.com/104712725/210695915-388a22ba-99af-4240-91ce-a9ff985434a4.png)

Immortal Pear v2
![грушаv3](https://user-images.githubusercontent.com/104712725/210889670-7225ca5b-ccdd-4658-9faf-3f0ece675ff5.png)
![грушаv31](https://user-images.githubusercontent.com/104712725/210889676-31baf28b-dbdc-4462-ae7b-51334044a2a9.png)
![грушаv3-export](https://user-images.githubusercontent.com/104712725/210889776-20b709ee-9709-4ed3-8e73-9f505c717dcb.png)
![груша прев v3](https://user-images.githubusercontent.com/104712725/210889977-c8c07905-9b24-4689-a54f-ed2f7f37fac8.png)

Immortal Pear v3
![pearv13](https://user-images.githubusercontent.com/104712725/211425998-7cc01c93-eab6-43d9-9d3d-6c9dea998921.png)
![pearv1313](https://user-images.githubusercontent.com/104712725/211426005-43b48541-b4c1-482b-ad89-8efc9a289962.png)
![pearv13exp](https://user-images.githubusercontent.com/104712725/211426015-f73c51eb-58ef-4f30-acec-4e589f7943d1.png)
![pear v13 exp](https://user-images.githubusercontent.com/104712725/211426019-045dbb80-fa33-4999-845f-a7dbf2829a63.png)

Immortal Pear v4
![pearv](https://user-images.githubusercontent.com/104712725/211617043-af899414-719e-43d0-8866-a031be3b2659.png)
![v16](https://user-images.githubusercontent.com/104712725/211617061-e30bef73-283a-45ca-b16d-d317f349225f.png)
![pearv-export](https://user-images.githubusercontent.com/104712725/211617071-ee05e9ef-ec01-41a5-9ba6-5cf5f5f0fb38.png)
![Sprite-0002](https://user-images.githubusercontent.com/104712725/211617084-cb74da45-9643-418f-9905-03c202cf27d8.png)

Golden Version
![голд](https://user-images.githubusercontent.com/104712725/211063805-f998cc67-da6f-472a-93cd-695fb2e4a9e7.png)
![голд2](https://user-images.githubusercontent.com/104712725/211063811-10211aa2-abdd-4f76-9092-c6ae0b2dffbb.png)
![голд-export](https://user-images.githubusercontent.com/104712725/211063864-579f86ff-0df2-4216-b157-7609cdcd4ec9.png)
![голд прев](https://user-images.githubusercontent.com/104712725/211063961-9b10e1f7-9a26-41ff-8f81-485480d4a0c8.png)

**Reply by Myth727** [reply]
Lovely, my only suggestion is aim to make it look a tad more golden! As per that is the main ingredient when minting :) otherwise looks great I think fire 
![Screenshot_20230105_024616_MetaMask](https://user-images.githubusercontent.com/96928557/210728295-e56ea069-2a99-48e0-bec3-6b99ba755a3c.jpg)

**Reply by Myth727** [reply]
Love it, happy to see version 3. I very much hope Adam and the team accept this one. Great work mate!

**Reply by rasheed19871** [reply]
i love sunflower land plz give me beta pass so instead of waiting i will do some testing and report bugs guys

**Comment by Whaitte** [comment]
**Lady Bug**

_V1_
![LadyBug.png](https://user-images.githubusercontent.com/104186329/210705480-dc71d113-a39b-4b3f-882a-fc34a5369bc0.png)
![LadyBug (2).png](https://user-images.githubusercontent.com/104186329/210705496-eaa0d7b7-a1e5-49e2-9e0c-471690c9230a.png)![LadyBug (1).png](https://user-images.githubusercontent.com/104186329/210705511-2a63ba80-5e64-4b0d-ab0e-b18067e53807.png)

_V2_
![LadyBug (3).png](https://user-images.githubusercontent.com/104186329/210705549-216b06ae-130d-4e7f-8018-32b1c26daed0.png)
![LadyBug (4).png](https://user-images.githubusercontent.com/104186329/210705562-a41ddfc5-e926-41ca-86b4-dd7adf2d7a2e.png)
![Ladybug S.png](https://user-images.githubusercontent.com/104186329/210705579-c490abad-82fc-46f7-b2aa-04155732e8f7.png)
![LadyBug (5).png](https://user-images.githubusercontent.com/104186329/210705588-d2b620a3-fb5a-439d-a388-fe7602f8b522.png)

_V3_
![LadyBug (8).png](https://user-images.githubusercontent.com/104186329/210705639-df1955c3-b826-4ef1-aa49-f0b3b10d5f96.png)
![LadyBug (7).png](https://user-images.githubusercontent.com/104186329/210705649-ff126ff3-4509-4b2d-a759-79dfc80a87df.png)
![LadyBug (9).png](https://user-images.githubusercontent.com/104186329/210705668-1cf5021e-b7d5-4cbb-a1d4-e7710799b0fe.png)

**Ladybug dress**

![LadyBug (14).png](https://user-images.githubusercontent.com/104186329/210705711-b5d58af5-e5ad-4897-bb61-72ef3736ba35.png)
![LadyBug (15).png](https://user-images.githubusercontent.com/104186329/210705722-0ca37e2c-205b-4e2d-a816-e082e905fb1e.png)
![LadyBug (12).png](https://user-images.githubusercontent.com/104186329/210705731-728e3978-68ca-47b2-8710-3fc0a8cb015f.png)
![LadyBug (13).png](https://user-images.githubusercontent.com/104186329/210705739-c39e24a9-1a6e-41dc-97f8-01c2e18830aa.png)
![LadyBug (10).png](https://user-images.githubusercontent.com/104186329/210705752-ad37f410-3c67-41e6-bb32-e524e068f5e4.png)
![LadyBug (11).png](https://user-images.githubusercontent.com/104186329/210705760-84258410-870b-46b3-8a3f-c0afbe33e7c7.png)

**ladybug shoes**

_V1_
![Ladybug (25).png](https://user-images.githubusercontent.com/104186329/210867626-7d1b1e4b-ba77-4e0b-8629-138ddec7f6c6.png)
![Ladybug (27).png](https://user-images.githubusercontent.com/104186329/210867644-d3c057ad-9e3b-45b5-924c-c90ee4cd1069.png)
_V2_
![Ladybug (23).png](https://user-images.githubusercontent.com/104186329/210867668-387a2ea8-ba08-4c9f-9e07-c31b9431dd13.png)
![Ladybug (24).png](https://user-images.githubusercontent.com/104186329/210867687-4926a75c-17ca-4d71-a9b5-433e0682208f.png)

**ladybug on trunk**
![Ladybug (1).png](https://user-images.githubusercontent.com/104186329/210894201-fc23d90c-b4d9-4f60-b048-8166b2f475c3.png)
![Ladybug (2).png](https://user-images.githubusercontent.com/104186329/210894224-42473b22-6b19-42d1-99bb-420a541654a8.png)
![Ladybug (3).png](https://user-images.githubusercontent.com/104186329/210894240-230f52d1-a404-4e34-81ad-273cef0fcf09.png)
![Ladybug (4).png](https://user-images.githubusercontent.com/104186329/210894256-f6b9d79e-3401-453e-b27a-5bb7c6b6d917.png)



Farm ID: 919
Discord: whaitte#3465

**Comment by mmaxxpeinn** [comment]
**Lady Bug**
![lbug](https://user-images.githubusercontent.com/120295844/210726597-db4d0543-efe6-4192-9623-939a9b380dbb.gif)
 minified:
![ladybug](https://user-images.githubusercontent.com/120295844/210726642-41db4dd4-ee08-49a8-9882-1864ffb7820f.gif)

Simplified Eye version
![bug2](https://user-images.githubusercontent.com/120295844/210729883-05eb3fb3-feed-4ec9-a667-23f22b9282b1.gif)
minified:
![lbug2](https://user-images.githubusercontent.com/120295844/210729188-377ff381-95aa-433f-a3ec-a7e7a9dd7a07.gif)

**Golden Pear**
![peaar](https://user-images.githubusercontent.com/120295844/210740584-e23d7a9f-8a01-4172-b3d3-bb4b937299ab.png)
minified (with/without case):
![pear](https://user-images.githubusercontent.com/120295844/210769122-4e3c2cd3-5d58-41a2-acad-6fdf9f3b8896.png) ![ppearr (1)](https://user-images.githubusercontent.com/120295844/210769157-8313ae0d-4f03-45a8-a578-4b91a3651e2f.png)

`FarmID: 82122`
`DC: mmaxxpeinn#9351`

**Reply by NomiceBinocle** [reply]
pear is incredible

**Comment by peperol** [comment]
lady bug near my tree house ;)

![Lady bug and tre house](https://user-images.githubusercontent.com/114354077/210832211-478b7ea9-d337-412b-a7b5-528fac307132.gif)

lady bug 
![Lady bug](https://user-images.githubusercontent.com/114354077/210832345-a6a35afd-8193-4339-b412-f517bcea0238.gif)

static lady bug
![Static lady bug (3)](https://user-images.githubusercontent.com/114354077/210832555-1beb22b6-e1ca-4da3-be86-1a1524424d24.png)

farm ID 145849

**Comment by Manu-M123** [comment]
**Immortal Pear**

![pera inmortalsola 500%](https://user-images.githubusercontent.com/120758758/210851662-9e500058-093c-48f1-9787-6d49f8db47a4.gif)
![pera inmortalsola 100%](https://user-images.githubusercontent.com/120758758/210851686-54154cef-6877-4df1-b306-6eda033a8802.gif)
 


ID: 78922
Discord: MANU m#4728

**Comment by sfwhat** [comment]
### Immortal Pear

![updown-exporttt](https://user-images.githubusercontent.com/95937808/210893544-f08ec9ab-65a7-4a13-a427-58586424c0cd.gif)


_Assets_
**Static**

v1![Sprite-0005](https://user-images.githubusercontent.com/95937808/210893570-8e2b362a-ef12-481a-9c0e-140ec202dc52.png)

v2![hoverss](https://user-images.githubusercontent.com/95937808/210893774-2debf9d1-3094-4d73-b6c8-764bf0e9be1b.png)


**Hovering**
![hover](https://user-images.githubusercontent.com/95937808/210893600-8bd448f4-c28c-413f-834b-29df8c144900.gif)


**Infinity**
![updowninf](https://user-images.githubusercontent.com/95937808/210893620-b3409c2a-a4b2-40c5-968f-76943ab0f0f3.gif)



### Ladybug

![Sprite-00099](https://user-images.githubusercontent.com/95937808/210901573-bf80bddf-8a24-4d14-8cb6-2b34f58653ba.gif)

Asset
![ladybugexport](https://user-images.githubusercontent.com/95937808/210901578-bfc6812c-ebc2-4706-b73e-50e24bca9326.gif)


---
SfWhat
Farm 6127

**Comment by esp123** [comment]
Lady Bug

![map-bug](https://user-images.githubusercontent.com/74337946/210940609-945de71a-1c6e-4f51-9363-68629581082b.png)
![lebugin](https://user-images.githubusercontent.com/74337946/210940626-b18c5244-941b-4569-ac78-d09f3341202c.gif)
![lubugin1](https://user-images.githubusercontent.com/74337946/210940639-a103a6ac-02b4-4a6e-9975-d5863d18763b.gif)

Immortal Pear and The Infinity Snake

![199652971-830c85ed-4993-4ba7-99ec-9e55504bd4cd](https://user-images.githubusercontent.com/74337946/211317381-712a2a8c-edaa-4052-b245-c253877e0690.png)
![immopear](https://user-images.githubusercontent.com/74337946/211316661-8b843d44-65d6-43dd-a778-5d75ff998758.gif)
![immopear1x1](https://user-images.githubusercontent.com/74337946/211316715-08acfab9-e588-4789-962c-08adef73aab8.gif)

Feel free to leave suggestions and comments 😉 
Discord: deeefault#0033

**Comment by Darkylian** [comment]
Hi all,
Instead of using the recurring symbol of "infinite" on this post. I have added a bit of variation by using "the ankh", the Egyptian symbol of immortality.

Static
![FULL_Immortality_Pear_Small_Display_Case](https://user-images.githubusercontent.com/64649046/211020322-85b2d358-d8a9-4fa2-a718-6b326a9f5d41.png)



mini
![miniFULL_Immortality_Pear_Small_Display_Case](https://user-images.githubusercontent.com/64649046/211019927-fe6fbd61-9746-45db-9959-82a6ada1f680.png)



Simple Animation
![SimpleAnim_Immortality_Pear_Small_Display_Case](https://user-images.githubusercontent.com/64649046/211019864-c00d258d-5ded-4e18-b87f-56ce86c923ab.gif)

mini
![miniSimpleAnim_Immortality_Pear_Small_Display_Case](https://user-images.githubusercontent.com/64649046/211019909-615b2f8d-1979-42f5-82f1-2b507910c980.gif)





Animated
![ANIMATED_Immortality_Pear_ANIM_Small_Display_Case](https://user-images.githubusercontent.com/64649046/211020100-4e4242ae-28ac-46c5-a3cc-ae0b4086f8b1.gif)


mini
![miniANIMATED_Immortality_Pear_ANIM_Small_Display_Case](https://user-images.githubusercontent.com/64649046/211020122-ecd61dfd-445f-4098-a6c7-e6e6d50c2a81.gif)


Discord: Darkylian#8376
Farm ID: 151673

**Comment by peperol** [comment]
She traveled for centuries she crossed mountains, deserts she crossed the seas and reached us ... she is the Highlander Pear the Immortal….

![EF8E47E7-9FEE-4EC8-B22D-64970833FBB9](https://user-images.githubusercontent.com/114354077/211069996-57695158-98f9-4156-9e17-d14c43f17530.png)



without case
![238A0DDA-7A33-4922-9574-401C268DE2DB](https://user-images.githubusercontent.com/114354077/211070085-a7b516a8-6687-4ea2-abbb-9f84ad00142f.png)


with case
![534F59CE-1C24-4627-BA92-A5B2ED97AD6B](https://user-images.githubusercontent.com/114354077/211070157-f8ad8ea5-1184-44f3-8921-915ec0371e1e.png)


farm ID 145849

**Comment by Whaitte** [comment]
_Preview 1_
![Ladybug preview 1.gif](https://user-images.githubusercontent.com/104186329/211168480-6ec70b86-9be2-4c11-8802-fca422359bec.gif)
_Minified_
![Ladybug V1.gif](https://user-images.githubusercontent.com/104186329/211168533-09eab97d-d04d-443c-93eb-13d7dcc5e162.gif)          ![Ladybug V2.gif](https://user-images.githubusercontent.com/104186329/211168536-1de33bd9-7c0f-4026-bce2-d4e46461a8c4.gif)           ![Ladybug V3.gif](https://user-images.githubusercontent.com/104186329/211168538-19615789-8317-463a-8eca-376ad30340f2.gif)           ![Ladybug V4.gif](https://user-images.githubusercontent.com/104186329/211168539-0db24c77-80a3-4be7-8265-d81fa3e94b65.gif)

_Preview 2_
![Ladybug preview 2.gif](https://user-images.githubusercontent.com/104186329/211168543-dacf6538-4fc8-4c41-9182-0784eca33839.gif)
_Minified_
![Ladybug V5.gif](https://user-images.githubusercontent.com/104186329/211168554-e324858d-4e55-453a-8b66-b198e695aaf4.gif)            ![Ladybug V6.gif](https://user-images.githubusercontent.com/104186329/211168556-53aa46bf-3922-4506-8d63-532430a17a4a.gif)           ![Ladybug V7 (1).gif](https://user-images.githubusercontent.com/104186329/211168560-c3bbe0be-1e1f-4d06-a5e4-6aa047484dd1.gif)            ![Ladybug V8.gif](https://user-images.githubusercontent.com/104186329/211168562-a071c5fb-0c33-4691-b0df-854322cb8602.gif)

_immortal pear_
![immortal pear.png](https://user-images.githubusercontent.com/104186329/211172295-ae15afea-7f25-4c69-b4eb-8c5949b52925.png)
_Minified_
![immortal pear (2).png](https://user-images.githubusercontent.com/104186329/211172310-8681beb8-b650-4292-a77d-5681e918fcf4.png)
![immortal pear (3).png](https://user-images.githubusercontent.com/104186329/211172350-819ef09e-b40a-48d8-ba0b-b1b46437eff6.png)


Farm ID: 919
Discord: whaitte#3465

**Comment by N1491** [comment]
Immortal Pear    
![pear](https://user-images.githubusercontent.com/121005517/211195213-97bf7ac3-2a05-4a05-b040-7514fc8f92d8.png)
![pear box](https://user-images.githubusercontent.com/121005517/211195219-2457aea5-e054-42a4-9a7d-42b4f70c4070.png)
![pear box(1)](https://user-images.githubusercontent.com/121005517/211195226-9e36f3bf-8053-45fe-8708-c41e19327a12.png)
FarmID 139032 Discord: PSGR | #139032

**Comment by N1491** [comment]
Lady Bug   
![bug](https://user-images.githubusercontent.com/121005517/211204114-5e7c7996-9248-43b2-91f4-7ade49467129.png)
![Lad Bug](https://user-images.githubusercontent.com/121005517/211204124-840d938d-fa8a-4573-a671-810f7e00baef.gif)
![Bug 1](https://user-images.githubusercontent.com/121005517/211204135-a94ccf79-73b5-4fde-a617-929874d07295.png)
 
 

![Lad Bug (2)](https://user-images.githubusercontent.com/121005517/211204300-d2e78982-5719-472a-aefc-04d7a0e8bc3c.gif)
FarmID 139032 Discord: PSGR | #139032

**Comment by N1491** [comment]
![Lad Bug (4)](https://user-images.githubusercontent.com/121005517/211204877-516aec45-1852-4b31-bb8e-62554cbbc4f4.gif)
FarmID 139032 Discord: PSGR | #139032    
![Lad Bug (5)](https://user-images.githubusercontent.com/121005517/211325310-1cab9914-1c01-4370-9b40-9be7cffda34b.gif)

**Comment by Vergelsxtn** [comment]
Ladybug

Preview:
![ladyb-prev](https://user-images.githubusercontent.com/99605039/211262828-af07a365-d9e7-4348-8b4d-0283882e35c3.gif)
Asset:
![lady-bug1](https://user-images.githubusercontent.com/99605039/211262838-ea18e638-5cd2-40a3-9686-23f63044bdef.gif)

__________
Vergelsxtn
Farm: 63

**Comment by NycksonV** [comment]
"A well-dressed traveler with his face covered appeared suddenly on my lands. Wherever he stepped, the ground turned golden and everything seemed too real to be just a dream. He approached my house and looked into my eyes, leaving a strange fruit on my porch. Then he said: 'Eat if you are worthy!'. In the same instant that he said this, he disappeared as quickly as he had appeared out of nowhere. His words were true, for this pear is certainly not just a common fruit!"

Here is a beautiful and mystical Immortal Pear!!!

![PeraGG](https://user-images.githubusercontent.com/118012559/211423437-e2d5ef8f-0a7c-4dc4-9826-043cb52adfb3.gif)
![PeraGG1](https://user-images.githubusercontent.com/118012559/211423440-1fd350e6-7c52-4197-bed4-bc2ed4c8e306.gif)
![PeraGG3](https://user-images.githubusercontent.com/118012559/211423444-6fe66d51-96ca-470d-9b57-e6cd67f9f4ec.gif)

![PeraG1](https://user-images.githubusercontent.com/118012559/211424125-a10da86e-9d7f-4948-b723-d34c73da70a7.gif)
![PeraG2](https://user-images.githubusercontent.com/118012559/211424129-60fdc642-65e5-4479-b590-64fcf85836f9.gif)
![PeraG3](https://user-images.githubusercontent.com/118012559/211424132-d36f1597-9559-4b35-b201-d78bfec57d05.gif)

![PeraImortal1](https://user-images.githubusercontent.com/118012559/211424146-8b3e6288-20bc-4c7e-90a8-bd657c4d6e89.gif)
![PeraImortal2](https://user-images.githubusercontent.com/118012559/211424148-323e536f-48a8-47c3-a970-b1ea154d7919.gif)
![PeraImortal3](https://user-images.githubusercontent.com/118012559/211424149-f5c83250-11e7-4291-a2e6-b047d571f73e.gif)


Land ID: 136998
Discord: PoroTampinha
Twitter: https://twitter.com/PoroTampinhaSFL

**Reply by MAISAbr** [reply]
Amei!!! top demais!!!!
